projects/js-packages/publicize-components/src/components/generated-image-preview/index.js

```diff
@@ -15,15 +15,40 @@ const getMediaSourceUrl = media => {
 	return media?.media_details?.sizes?.large?.source_url || media?.source_url;
 };
 
-export const calculateImageUrl = ( imageType, customImageId, featuredImageId, getMedia ) => {
+const getImageId = ( imageType, customImageId, featuredImageId, defaultImageId ) => {
+	if ( imageType === 'custom' && customImageId ) {
+		return customImageId;
+	}
+
+	if ( imageType === 'default' && defaultImageId ) {
+		return defaultImageId;
+	}
+
+	if ( imageType === 'featured' && featuredImageId ) {
+		return featuredImageId;
+	}
+
+	return featuredImageId || defaultImageId;
```

Comment:
IMHO, it should be this

```suggestion
	if ( imageType === 'custom' ) {
		return customImageId;
	}

	if ( imageType === 'default' ) {
		return defaultImageId;
	}

	if ( imageType === 'featured' ) {
		return featuredImageId;
	}

	return featuredImageId || defaultImageId;
```